cookbook/case_studies/ml_training/spark_horovod/README.rst

```diff
@@ -1,6 +1,6 @@
 .. _spark_horovod:
 
-Forecast Sales Using Rossmann Store Sales Data Using Horovod and Spark
+Forecast Sales Using Rossmann Store Sales Data With Horovod and Spark
```

Comment:
```suggestion
Forecasting Rossman Store Sales with Horovod and Spark
```

(the "with" is a preposition, and so should be lower case)

Reply:
It depends. In [APA](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) style title case, "with" has to be "With", whereas in Chicago style title case, "with" has to be "with".